app/celery/tasks.py

```diff
@@ -305,11 +301,7 @@ def save_smss(self, service_id: Optional[str], signed_notifications: List[Signed
     check_service_over_daily_message_limit(KEY_TYPE_NORMAL, service)
     current_app.logger.info(f"Sending following sms notifications to AWS: {notification_id_queue.keys()}")
     for notification_obj in saved_notifications:
-        try:
-            check_service_over_daily_sms_limit_and_warn(KEY_TYPE_NORMAL, service)
-        except (LiveServiceTooManySMSRequestsError, TrialServiceTooManyRequestsError):
-            # if notification would put service over limit, don't add it to the queue
-            continue
+        send_warning_sms_limit_emails_if_needed(notification_obj.service)
```

Comment:
Really this shouldn't be needed when we start incrementing in API (i.e. not in celery), as the email will be triggered when the POST request is made.

Reply:
Sounds good, I will move this upstream